v4/source/users.yaml

```diff
@@ -1230,7 +1230,44 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+  
+  /users/tokens:
+    get:
+      tags:
+        - users
+      summary: Get all user access tokens for all users
```

Comment:
How about just 'Get user access tokens'?

Reply:
On phone but I think that's not correct. It should be: 'Get users' access tokens' if anything. One is getting all user access tokens for all users.